desktop_version/CMakeLists.txt

```diff
@@ -145,9 +145,10 @@ if (DEFINED SDL2_INCLUDE_DIRS AND DEFINED SDL2_LIBRARIES)
 else()
 	# Only try to autodetect if both SDL2 variables aren't explicitly set
 	find_package(SDL2 CONFIG)
+	find_package(sdl2-mixer CONFIG)
 	if (TARGET SDL2::SDL2)
 		message(STATUS "Using TARGET SDL2::SDL2")
-		target_link_libraries(VVVVVV SDL2::SDL2 SDL2_mixer)
+		target_link_libraries(VVVVVV SDL2::SDL2 SDL2::SDL2main SDL2::SDL2_mixer)
```

Comment:
Because of the above issue, the SDL2_mixer change here will probably have to be reverted, but the SDL2main part is definitely a bug, so good catch there.

Reply:
Do you know if thats always required or vcpkg specific too?